cypress/e2e/shared/checks.test.ts

```diff
@@ -10,14 +10,17 @@ describe('Checks', () => {
     cy.flush()
 
     cy.signin().then(() => {
-      cy.writeData([`${measurement} ${field}=0`, `${measurement} ${field}=1`])
-
       // visit the alerting index
-      cy.get('@org').then(({id}: Organization) =>
+      cy.get('@org').then(({id: orgID, name: orgName}: Organization) => {
+        cy.writeData(
+          [`${measurement} ${field}=0`, `${measurement} ${field}=1`],
+          orgName,
+          'defbuck'
```

Comment:
is this hard coded bucket name supposed to be here?

Reply:
there are many instances of hard coded bucket name. I don't want to clean up e2e tests for you. You are welcome to pair with me on this.

Reply:
lets just say for the lines you're adding, why not use `cy.get('@bucket')`?

Reply:
Should this get bucket name from @bucket or Cypress.env('bucket') ? I don't want to make these decisions for the UI team.

Reply:
what was your reasoning for replacing the `Cypress.env('org')` with `@org`? maybe we can use that to guide the discussion

Reply:
getting from @bucket

Reply:
org name is different for each user.